solidity/truffle-examples/url-requests/test/url-requests-tests.js

```diff
@@ -101,228 +98,176 @@ contract('Oraclize Example using Truffle', async accounts => {
         contract._jsonInterface,
         contract._address
       )
-      urlReq[4] = { methods, events }
+      URL_REQUEST_CONTRACTS[4] = { methods, events }
       const { events: txEvents } = await methods
         .requestCookies()
         .send({
-          from: addr,
-          gas: gasAmt
+          from: ACCOUNT_ZERO,
+          gas: GAS_AMOUNT
         })
       const description = txEvents.LogNewOraclizeQuery.returnValues.description
       assert.strictEqual(
         description,
-        'Oraclize query was sent, standing by for the answer...',
-        'Oraclize query incorrectly logged!'
+        QUERY_SENT_STRING,
       )
     })
 
     it('Should log a failed second request for custom headers due to lack of funds', async () => {
-      const { events } = await urlReq[0]
+      const { events } = await URL_REQUEST_CONTRACTS[0]
         .methods
         .requestCustomHeaders()
         .send({
-          from: addr,
-          gas: gasAmt
+          from: ACCOUNT_ZERO,
+          gas: GAS_AMOUNT
         })
       const description = events.LogNewOraclizeQuery.returnValues.description
       assert.strictEqual(
         description,
-        'Oraclize query was NOT sent, please add some ETH to cover for the query fee',
-        'Oraclize query incorrectly logged!'
+        QUERY_NOT_SENT_STRING
       )
     })
 
     it('Should log a failed second basic auth request due to lack of funds', async () => {
-      const {events } = await urlReq[1]
+      const {events } = await URL_REQUEST_CONTRACTS[1]

```

Comment:
```suggestion
      const { events } = await URL_REQUEST_CONTRACTS[1]
```

stylistic correction

Reply:
:rofl: Good spot eagle eyes. I shall commit your suggestion and take my de-merit with good grace.